packages/kbn-expandable-flyout/src/provider.tsx

```diff
@@ -70,6 +82,10 @@ interface ExpandableFlyoutProviderProps {
    * Url storage syncs current values straight to the browser query string.
    */
   storage?: ExpandableFlyoutStorageMode;
+  /**
+   *
+   */
+  urlKey: string;
```

Comment:
@lgestc @christineweng while updating the README, I just thoughts that we could potentially simplify the props that we pass in. Currently we pass both `storage` and `urlKey` which is a bit redundant. I think I could easily have a single property here. Let me know if you think it's worth doing in the PR or we should do it in a follow up!

Reply:
I actually decided to implement this. I updated the PR description accordingly

Reply:
@PhilippeOberti sounds good to me, in addition to "don't use memory", I would just suggest including documentation on how it impacts the storage